### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by xisd**

### CODE REVIEW

**Comment by xisd** [comment]
I changed the api.lua because french translated string were longer and didn't fit
(made notification box larger and replaced label for title by textarea to enable line break )

**Comment by sofar** [comment]
This needs manual squashing by @rubenwardy or @Wuzzy2 - should end up with 2 commits (layout changes and translation file).

:+1: on content. looks fine to me.

File: api.lua [hunk 1]
```diff
@@ -367,9 +367,8 @@ function awards.getFormspec(name, to, sid)
 			if item.got then
 				status = S("%s (got)")
 			end
-			formspec = formspec .. "label[1,2.75;" ..
-				string.format(status, minetest.formspec_escape(title)) ..
-				"]"
+			formspec = formspec .. "textarea[0.5,2.7;4.8,1.45;;"..string.format(status, minetest.formspec_escape(title))..";]"
```
Comment (rubenwardy) [inline api.lua]: I'd prefer if this line was wrapped like the previous version

File: locale/fr.txt [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+You have disabled your achievements. = Vous avez désactivé vos réussites.
+You have enabled your achievements. = Vous avez activé vos réussites.
+You have not unlocked any awards. = Vous n'avez encore débloqué aucun trophé.
+You've disabled awards. Type /awards enable to reenable. = Vous avez désactivé les trophés. Tapez '/awards enable' pour les réactiver.
+You’re a copper = Faites Jouer les Cuivres
+[c|clear|disable|enable] = [c|clear|disable|enable]
+OK = OK
+Error: No awards available. = Erreur : Aucun trophé disponible
+
+#### nodes.lua ####
```
Comment (rubenwardy) [inline locale/fr.txt]: Not here yet

File: locale/fr.txt [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+(Secret Award) = (Trophé Secret) 
+<achievement ID> = <réussite ID>
+<name> = <nom>
+A Cat in a Pop-Tart?! = Un Chat dans un Pop-Tart ?!
+Achievement Unlocked! = Réussite Débloquée !
+Achievement Unlocked: = Réussite Débloquée :
+Achievement Unlocked: %s = Réussite Débloquée : %s
+Achievement not found. = Réussite non trouvée.
+All your awards and statistics have been cleared. You can now start again. = Tous vos trophés et statistiques ont été réinitialisés. Vous pouvez recommencer a nouveau.
+Aspiring Farmer = Apprenti.e. Fermière/Fermier
```
Comment (rubenwardy) [inline locale/fr.txt]: Apparently Apprenti.e. is wrong - did you mean Apprenti?

http://www.wordreference.com/fren/Apprenti

File: locale/fr.txt [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+Awards = Trophés
+Backpacker = Randonneuse/Randonneur
+Bankier = Banquière/Banquier
+Bricker = Briqueuse/Briqueur
+Build a Cave = Construit une grotte
+Castorama = Castorama
```
Comment (rubenwardy) [inline locale/fr.txt]: is this a real string?

File: locale/fr.txt [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+Craft 10 furnaces. = Construit 10 fours.
+Craft 10 mese lamps. = Fabrique 10 lampes de mese.
+Craft 100 obsidian bricks. = Confectionne 100 briques d'obsidienne.
+Craft 100 sandstone bricks. = Confectionne 100 briques de roche de sable.
+Craft 100 sticks. = Confectionne 100 bâtons.
+Craft 100 white dyes. = Prépare 100 teintures blanches. 
+Craft 14 vessels shelves. = Fabrique 14 vesseliers.
+Craft 15 chests. = Fabrique 15 coffres.
+Craft 15 fancy beds. = Consrtuit 15 lits chics.
```
Comment (rubenwardy) [inline locale/fr.txt]: this comes up as spelled incorrectly as well

File: locale/fr.txt [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+Craft 200 brick blocks. = Confectionne 200 blocs de brique.
+Craft 200 stone bricks. = Confectionne 200 briques de pierre.
+Craft 24 gold block stairs. = Consrtuit 24 escaliers en or.
+Craft 250 white wool. = Confectionne 250 blocs de laine blanche.
+Craft 3,200 stone bricks. = Confectionne 3,200 briques de pierre.
+Craft 30 locked chests. = Fabrique 30 coffres
```
Comment (rubenwardy) [inline locale/fr.txt]: this doesn't mean locked, afaict

**Comment by sofar** [comment]
No response from submitter to fix issues found. Will consider closing this if nobody can fix the PR.

File: locale/fr.txt (hunk 2 above)
Comment (xisd) [inline locale/fr.txt]: removed this part

File: locale/fr.txt (hunk 6 above)
Comment (xisd) [inline locale/fr.txt]: corrected

File: locale/fr.txt (hunk 5 above)
Comment (xisd) [inline locale/fr.txt]: yep corrected

File: locale/fr.txt (hunk 4 above)
Comment (xisd) [inline locale/fr.txt]: Wich one  ?

File: locale/fr.txt (hunk 3 above)
Comment (xisd) [inline locale/fr.txt]: it spells Apprenti for a man and Apprentie for a woman
Apprenti.e. is an other way or writting Apprenti(e)

File: api.lua (hunk 1 above)
Comment (xisd) [inline api.lua]: Do you means splitting the line in the file or changing the formspec ?

**Comment by xisd** [comment]
crap... I wanted to merge the branch containing translation with into the one containg award board and I endend up doing the opposite. Is there a way to undo this ?! 
Might I as welll delete everything and re-fork properly ?

**Comment by xisd** [comment]
This PR is no longer valid : I deleted and re-forked
There is now one for the french strings, and an other one for the formspec edit,
and the board node is now a separate mod (depedant to this one )
(https://github.com/xisd/awards_board)

**Pull request closed without merging**